chronosphere/resource_notification_policy.go

```diff
@@ -233,23 +233,17 @@ func (npr *notificationPolicyResourceMeta) resourceNotificationPolicyCustomizeDi
 		return err
 	}
 
-	// Only one of route or rule can be set.
-	if len(policy.Route) > 0 && len(policy.Rule) > 0 {
-		return errors.New("cannot set both `route` and `rule`, update `rule` blocks to use `route` instead.")
-	}
-
-	// At least one-of route, rule, or override must be set.
-	// Note: we don't mention rule in the error, since it's deprecated (route should be used instead).
-	if len(policy.Route) == 0 && len(policy.Rule) == 0 && len(policy.Override) == 0 {
+	// At least one-of route, or override must be set.
+	if len(policy.Route) == 0 && len(policy.Override) == 0 {
 		return errors.New("specify at least one `route` or `override`")
 	}
 
-	// rule/route blocks are merged, which is simple for notifiers as the list can be appended.
+	// route blocks are merged, which is simple for notifiers as the list can be appended.
 	// repeat_interval cannot be easily merged, so ensure that blocks with the same severity
 	// have the same repeat_interval.
-	validateRouteElem := func(rules []intschema.NotificationRoute) error {
+	validateRouteElem := func(routes []intschema.NotificationRoute) error {
```

Comment:
nit: can inline this logic now, previously we ran it twice for rules/routes, now we don't need to do that, so we can simplify